src/rpc/blockchain.cpp

```diff
@@ -131,7 +131,7 @@ struct MinterInfo {
         CKeyID minter;
         block.ExtractMinterKey(minter);
         auto id = view.GetMasternodeIdByOperator(minter);
-        if (!id) return {};
+        if (!id && block.deprecatedHeight != 0) return {};
```

Comment:
Use blockindex->nHeight != 0 instead for more correctness to check for DVM genesis block